openassessment/xblock/grade_mixin.py

```diff
@@ -49,17 +49,20 @@ def render_grade(self, data, suffix=''):
         status = workflow.get('status')
 
         # Default context is empty
-        context = {}
+        context = {'xblock_id': self.get_xblock_id()}
 
+        assessment_steps = self.assessment_steps
         # Render the grading section based on the status of the workflow
         try:
             if status == "cancelled":
                 path = 'openassessmentblock/grade/oa_grade_cancelled.html'
-                context = {'score': workflow['score']}
+                context['score'] = workflow['score']
             elif status == "done":
                 path, context = self.render_grade_complete(workflow)
             elif status == "waiting":
-                path, context = 'openassessmentblock/grade/oa_grade_waiting.html', {}
+                if "staff-assessment" in assessment_steps:
+                    context['is_waiting_staff'] = "is--waiting--staff"
```

Comment:
Could you add a comment in the code explaining that you need this information to know where focus should next move?